packages/platforms/react-native/lib/persistence/file-native.ts

```diff
@@ -0,0 +1,52 @@
+import type { Encoding, Spec } from '../NativeBugsnagPerformance'
+import { NativeModules } from 'react-native'
+
+const LINKING_ERROR =
+  'The package \'BugsnagReactNativePerformance\' doesn\'t seem to be linked.'
+
+// @ts-expect-error check if turbomodule is enabled
+const isTurboModuleEnabled = global.__turboModuleProxy != null
+
+const BugsnagFileAccessModule = isTurboModuleEnabled
+  ? require('./NativeBugsnagPerformance').default
+  : NativeModules.BugsnagReactNativePerformance
```

Comment:
this shouldn't be necessary - there's a file called `native.ts` that handles this logic, so you can just do:

```ts
import NativeBugsnagPerformance from '../native'
```

Reply:
removed